### PR TITLE
Caleb/pixytoo

### PR DESCRIPTION
**Pull request by Tigerbytes1775**

### CODE REVIEW

**Comment by clebio** [comment]
Any idea on these conflicts, @wentzt ?

File: src/org/usfirst/frc/team1775/robot/OI.java [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
-import org.usfirst.frc.team1775.robot.commands.Solenoid;
 
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-	public static Joystick driverJoystick;
-	public static Joystick mechanismJoystick;
+	//// CREATING BUTTONS
```
Comment (jforevers) [inline src/org/usfirst/frc/team1775/robot/OI.java:10]: why not make this a multiline comment? 
/*
*
*/

and for all multiline comments below

File: src/org/usfirst/frc/team1775/robot/subsystems/PixyException.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.usfirst.frc.team1775.robot.subsystems;
+
+
```
Comment (jforevers) [inline src/org/usfirst/frc/team1775/robot/subsystems/PixyException.java:3]: Nit. extra new lines

File: src/org/usfirst/frc/team1775/robot/subsystems/PixyException.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+	public class PixyException extends Exception{
+		public PixyException(String message){
+			super(message);
+		}
+}
+
```
Comment (jforevers) [inline src/org/usfirst/frc/team1775/robot/subsystems/PixyException.java:10]: unneeded new lines

File: src/org/usfirst/frc/team1775/robot/subsystems/TestSPI.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	public TestSPI() {
+		pixy =  new SPI(SPI.Port.kOnboardCS0);
+		pixy.setMSBFirst();
+		pixy.setChipSelectActiveLow();
+		pixy.setClockRate(1000);
+		pixy.setSampleDataOnFalling();
+	    pixy.setClockActiveLow();
+	}
+	
+	public void doSomething() {
```
Comment (jforevers) [inline src/org/usfirst/frc/team1775/robot/subsystems/TestSPI.java:17]: could add some javadoc with descriptions of the methods

File: src/org/usfirst/frc/team1775/robot/subsystems/VisionProcessor.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+		}
+			
+		pPackets.put("x", Xs);
+		pPackets.put("y", Ys);
+		pPackets.put("w", Ws);
+		pPackets.put("h", Hs);
+		
+		return pPackets;
+	}
+
```
Comment (jforevers) [inline src/org/usfirst/frc/team1775/robot/subsystems/VisionProcessor.java:86]: extra new lines

**Comment by clebio** [comment]
Thanks for the comments. I'm closing this PR since I don't think we should merge this to master yet (not clear whether/how/when it will be used, yet).

**Pull request closed without merging**